utils/scarb-test-support/src/project_builder.rs

```diff
@@ -95,6 +103,26 @@ impl ProjectBuilder {
         self.dep("starknet", Dep.version(CAIRO_VERSION))
     }
 
+    pub fn readme(mut self, readme: impl ToString) -> Self {
```

Comment:
Did we end up using those anywhere?

Reply:
My bad, not used anywhere since we're using `--no-metadata` flag, thanks!